database/generator.js

```diff
@@ -64,12 +65,12 @@ let multipleImg = function() {
 
 let categoryName = function() {
   let output = [];
-  
+
   //random number between 1-7 for for loop upper limit
   for (let i = 1; i < getRandomInt(1, 4); i++) {
-    output.push({ 
+    output.push({
       categoryIterator: i,
-      category: faker.lorem.word(), 
+      category: faker.lorem.word(),
       options: optionCategories(counter)
```

Comment:
If it doesn't concern you, perhaps look into object referencing instead of instantiating every obj with it's own version. 

Pass the same data around.

Reply:
generator.js is legacy. my code is in productGenerator.js - it creates CSV files that will be copied to DB directly.

Reply:
COOL COOL